architecture/kubernetes_infrastructure.adoc

```diff
@@ -9,30 +9,43 @@
 
 toc::[]
 
-== Master
+Kubernetes is a system for managing containerized applications across a set of containers or hosts, providing deployment, maintenence, and application-scaling mechanisms. Docker is used to package, instantiate, and run containerized applications. 
 
-The host(s) containing the master components (API Server, Controller Manger Server, etcd, etc)
+A Kubernetes cluster involves a master and a set of nodes. 
```

Comment:
@bfallonf Would this be better stated as "A Kubernetes cluster consists of a master and a set of nodes"?